### ISSUE TITLE
Multiple result set support

### ISSUE THREAD

**Issue by Paddy-GitHub** [tmenier/AsyncPoco#55]
[Petapoco](https://stackoverflow.com/questions/19576370/petapoco-multiple-result-set-support) and [Dapper](https://github.com/StackExchange/Dapper#multiple-results) support database results with multiple sets.

Would you like me to port the multiple result set support feature from PetaPoco to AsyncPoco?

**Comment by tmenier** [comment]
Yes, that would be great! The main thing I want to emphasize is to please include tests in your PR. They should follow the existing patterns used throughout the test suite where the feature is automatically tested against all db platforms. (Or those that support it anyway, if it's not applicable to all.)

Thanks!

**Comment by SashaBeluj** [comment]
Supporting multiple result sets is an important feature we need and it would be great to see it in AsyncPoco.

**Comment by AlexFree** [comment]
Hi guys, when do you plan to release multiple query results?

**Comment by Paddy-GitHub** [comment]
Hi,
I don't have a timescale I'm afraid. I have a few other urgent jobs on
before I can look at this so it will be some time yet.
Thanks
